Makefile

```diff
@@ -133,9 +134,48 @@ build-darwin:
 	  $(MAKE) build
 
 build-linux:
-	SC_GOOS=linux SC_GOARCH=amd64 BINDIR=bin/darwin_amd64 DOCKER=1 \
+	SC_GOOS=linux SC_GOARCH=amd64 BINDIR=bin/linux_amd64 DOCKER=1 \
 	  $(MAKE) build
 
+images: build-linux $(shell find pkg contrib -name Dockerfile)
```

Comment:
I wasn't sure how often people use this stuff, but I can pull them out into separate targets.